src/elements/pill.svelte

```diff
@@ -0,0 +1,35 @@
+<svelte:options immutable tag='v-pill' />
+
+<script lang='ts'>
+
+import { dispatcher } from '../lib/dispatch';
+import { addStyles } from '../lib/index';
+import { htmlToBoolean } from '../lib/boolean';
+
+export let value = ''; 
+export let removable = 'true';
+let isRemovable: boolean;
+$: isRemovable = htmlToBoolean(removable, 'removable');
+
+
+const dispatch = dispatcher();
+addStyles();
+
+const handleRemove = () => {
+  dispatch('remove', { value });
+};
+
+</script>
+
+<!-- svelte-ignore a11y-click-events-have-key-events -->
```

Comment:
nit: we're not super focused on accessibility just yet so maybe not a big deal right now, but for accessibility reasons, a user should be able to tab to the x on this pill and press enter to delete this pill. I think this is copy and pasted, but something to consider for the future